diff --git a/demo/demo-1.ts b/demo/demo-1.ts
index ffc09dbd..79ca0bae 100644
--- a/demo/demo-1.ts
+++ b/demo/demo-1.ts
@@ -16,7 +16,7 @@ const holdCheckbox = getInput("hold");
 const muteCheckbox = getInput("mute");
 
 // WebSocket Server URL
-const webSocketServer = "wss://edge.sip.onsip.com";
+const webSocketServer = `wss://${window.location.hostname}:9876`;
 serverSpan.innerHTML = webSocketServer;
 
 // Destination URI
diff --git a/demo/demo-users.ts b/demo/demo-users.ts
index a1591244..f47329df 100644
--- a/demo/demo-users.ts
+++ b/demo/demo-users.ts
@@ -33,8 +33,8 @@ const domain = "sipjs.onsip.com";
 
 export const nameAlice = "Alice";
 export const uriAlice = "sip:alice." + token + "@" + domain;
-export const webSocketServerAlice = "wss://edge.sip.onsip.com";
+export const webSocketServerAlice = `wss://${window.location.hostname}:9876`;
 
 export const nameBob = "Bob";
 export const uriBob = "sip:bob." + token + "@" + domain;
-export const webSocketServerBob = "wss://edge.sip.onsip.com";
+export const webSocketServerBob = `wss://${window.location.hostname}:9876`;
